EpikCord/__main__.py

```diff
@@ -8,10 +8,17 @@
 __version__ = "0.5.2"
 _bot_tmplate = """#!/usr/bin/env python
 from EpikCord import Client, Intents
+from config import token
+class [name]Client(Client):
 
+    def __init__(self):
+        super().__init__()
```

Comment:
This part is useless as \_\_init\_\_ will be automatically called on parent if the child class don't have that dunder

Reply:
You need to set intents and a token...